meshtastic/interfaces/ble/interface.py

```
@@ -1451,28 +1991,48 @@ def _establish_and_update_client(
+                self.address,
+                self._register_notifications,
+                # Defer _connected() publication until this interface has committed
+                # ownership of the new client and the post-connect stale/closing
+                # checks have passed.
+                lambda: None,
+                self._on_ble_disconnect,
+                pair_on_connect=pair_on_connect,
+                connect_timeout=connect_timeout,
+            )
+        except ValueError as exc:
+            raise self.BLEError(f"invalid connect_timeout: {exc}") from exc
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)